management.md

```diff
@@ -126,5 +126,14 @@
     the help of test cases which can indicate whether the issue(s) have been
     solved or they are still present.
 
-- How will your team know when it is finished with the
-  implementation of the software project?
+- How will your team know when it is finished with the implementation of the
+  software project?
+
+  Our team will know when it is finished with the implementation of the software
+  project because of three reasons. First, our software project will not only
```

Comment:
"because of three reasons" phrasing is very basic - maybe say something like: "when the following is complete"